trading_calendars/exchange_calendar_cmes.py

```diff
@@ -40,7 +40,7 @@
 )
 
 
-class CMEExchangeCalendar(TradingCalendar):
+class CMESExchangeCalendar(TradingCalendar):
     """
     Exchange calendar for CME
```

Comment:
Should we update this with a (CMES)? We appear to have done that for the other calendars.